webpack.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
   devServer: {
     contentBase: path.join(__dirname, '/src/'),
     disableHostCheck: true,  //Allow access using custom host names, e.g. local.zooniverse.org:3998, which is necessary for testing certain API calls.
-    https: true,
+    // https: true,
```

Comment:
Not blocking: Should leave this as https by default and change it locally if we need to? 
FWIW i had to disable https for local http API integration dev / testing. Once we deploy the labs-datasette-api it'll be on https so this setting won't impact.

Reply:
Yup, I've reverted this back to `https: true`